jujugui/static/gui/src/test/test_env_api.js

```diff
@@ -3187,10 +3187,17 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
       });
     });
 
-    describe('addKeys', function() {
+    describe('_addKeys', function() {
```

Comment:
As we don't ever call the private method directly these tests should call the public method still as with the rest of the tests in this suite.